srcCpp/RTIDDSImpl.h

```diff
@@ -125,11 +87,21 @@ class RTIDDSImpl : public IMessaging
     bool configureDomainParticipantQos(DDS_DomainParticipantQos &qos);
 
   #ifndef RTI_MICRO
+
+    /**
```

Comment:
/*
 * @brief This function calculates the overhead bytes added by all the
 * members on the TestData_* type, excluding the content of the sequence.

Reply:
This is the way to mark a Doxygen comment.